1.Bash-Scripts/cpu_memory_usage

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#
+#Get the CPU and Memory usage
+
+#Get the overall CPU usage percentage by subtracting the idle time from 100
+cpu_usage=$(top -b -n1 | awk '/%Cpu/ { print 100-$8}')
```

Comment:
Here you are extracting following line from `top` output:
```
%Cpu(s):  0.0 us,  0.0 sy,  0.0 ni, 100.0 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
```
and you are getting the value before `id`.
> **id (Idle)**: Represents the percentage of time the CPU was not used by any process. A very low idle value indicates the system is heavily loaded.

This does not give us the actual CPU usage.
You should extract value next to `us` or `sy`.
> **us (User CPU time)**: This shows the percentage of CPU utilization that occurred while executing at the user level (applications). High user CPU time can indicate that your applications or processes are busy.

> **sy (System CPU time)**: Represents the percentage of CPU utilization that occurred in the kernel or system level. A high system time can indicate that the OS is busy with system-level tasks, possibly due to I/O waits, system calls, or other kernel operations.

So, it will be enough if we just print following:
```
top -b -n1 | awk '/%Cpu/ { print $2}' ## for user cpu
top -b -n1 | awk '/%Cpu/ { print $4}' ## for system cpu
```

> **HINT**: You can append "%" also during `awk` with `awk {print $2 "%"}`.